### ISSUE TITLE
In-App Browser doesn't show drop down items with certain Base Themes

### ISSUE THREAD

**Issue by MustardCat** [ccrama/Slide#2257]
Slide version: 5.5.1
Android version: 7.0 (and 5.1)

The In-App Browser doesn't show drop down items when certain base themes (all except Light) are enabled.

Base themes with the issue: Sepia, Dark Blue, Dark, AMOLED w/ Contrast, AMOLED, Red Shift

Original post of bug (with multiple confirmations): https://www.reddit.com/r/slideforreddit/comments/535oiy/bug_in_app_browser_doesnt_show_drop_down_items/

Screenshots:
5.1 - http://i.imgur.com/A8g1eM4.jpg
7.0 - http://i.imgur.com/Ha3qve6.png

Example URL to test drop down: https://www.microsoft.com/en-us/software-download/windows10ISO

Extra info: I'm 99% sure this was fixed in 5.5Alpha3 but came back in Alpha3.5.

**Comment by ccrama** [comment]
Unfortunately that is unfixable without making the top toolbar items black

**Comment by MustardCat** [comment]
Unless I'm misunderstanding - the drop down items do not have this issue (and appear correctly) when the base theme is set to Light.  The top bar text is also white for the Light theme so wouldn't that theme also have this issue?

If you are talking about the drop down items (Search, Refresh, etc), the Sepia base theme uses a dark grey color for those (so there shouldn't be an issue with the Sepia theme but there is).

**Comment by ccrama** [comment]
It appears correctly because the font color is set to the toolbar color, but the background color is always white. There is not really a way around that, unfortunately, so either the toolbar is incorrect or the page items are incorrect

**Comment by MustardCat** [comment]
Oh, I understand what are you saying now.  I had to recompile the alphas until I found the one where the in-page drop down items appeared to understand (it was Alpha 2).  In that Alpha 2, the toolbar drop down items were black for every theme (even the dark themes).

I did notice something interesting in Slide 5.5.1 though.  On my tablet, running 5.1, the Sepia theme correctly shows the in-page drop down items.  On my phone, running 7.0, the drop down items are white.

It's a really small issue though (and considering the other themes show white text, not a big deal to get fixed) so I'll close this.
